how-to/reports/comments-and-tags.md

```diff
@@ -11,29 +11,39 @@ This section describes how to add comments and tags to Reports and individual Sn
+
+To include your comments in the generated documentation navigate to the __Backstage__ and on the __Configuration__ screen and select __Options__. Under __Report Options__ check the __Include comments in documentation__ option. Your comments will now be visible in the generated documentation. 
 
-To include your comments in the generated documentation navigate to the __Backstage Configuration Screen__ and select __Options__. Under __Report Options__ check the __Include comments in Documentation__ option. Your comments will now be visible in the generated documentation. 
 
 ### __Tags__
 
 In the Permissions Explorer, you can find another useful feature - adding tags to Securable Objects.
 
 Tags can be added Live but also in all created Snapshots.
 
-To add a tag navigate to the top menu's __Tags__ tab, select your target Securable Object in the __Permissions Explorer__ and select the __Add Tag__ button. In the same tab, you can also find the __Delete Tag__ button to remove the tag from a selected Object. 
+To add a tag navigate to the top menu's __Tags__ tab under __Data__ category, select your target Securable Object in the __Permissions Explorer__ and select the __Add Tag__ button. In the same tab, you can also find the __Delete Tag__ button to remove a tag from the selected Object. 
+
+__Add Tag__ and __Delete Tag__ options are also available in the context menu which appears if you right-click a Securable Object in the tree view.
+
+To view all created tags navigate to __Snapshots__ section and select the __Tagged Objects__ report. 
+Information about all tagged objects is displayed in the __Object__, __Object Type__ and __Object Url__ columns and by default grouped by __Tag__.  The __Tagged Objects__ report can also be accessed directly from the __Permissions Explorer__ by selecting the __Go to Report__ option in the top menu's __Tags__ tab. 
+
+The __Tagged Objects__ report can easily be exported by selecting the __Export as PDF/XLSX/DOCX__ option from the __Home__ tab in the top menu.
+
+Each of the tagged objects can easily be accessed by selecting the appropriate row in the __Tagged Objects__ report and clicking the __Go to Object__ button in the top menu __Tags__ tab.
 
-To view all created tags navigate to __Snapshots__ section and select the __Tagged Objects__ overview. 
-Information about the tagged objects is displayed in the __Object__, __Object Type__ and __Object Url__ columns and by default grouped by __Tag__.  
\ No newline at end of file
+Another great thing is the possibility to __search the Securable Objects by tags__. In the __Permissions Explorer__ simply type the tag name into the __Search__ text field. All Securable Objects tagged with the inputed text are displayed in a dropdown list. When you select a single object from the list, the object also becomes selected in the tree view and the permissions are displayed in the grid. 
```

Comment:
Another _**upside of using tags / them**_ is the possibility to __search the Securable Objects by tags__.
it may sound weird because you mention tags again at the end

Reply:
maybe:
Another upside of tagging is the possibility to search the Securable Objects by tags.
or
Another upside of using the tag feature is the possibility to search the Securable Objects by tags.

Reply:
hmm your choice :)